frontend/src/types/index.ts

```diff
@@ -28,3 +28,19 @@ export type TaskType = {
 };
 
 export type ScreenModeType = 'DESKTOP' | 'MOBILE';
+
+export type UserImageType = {
+  '160w': string;
+  '240w': string;
+  '320w': string;
+  '480w': string;
+  fallback: string;
+};
```

Comment:
```suggestion
export type UserImageType = '160w' | '240w' | '320w' | '480w' | { fallback: string };
```

'160w' 같은 값들은 변수가 아닌 상수인데, string 타입이 아닌 상수 자체로써 타입 선언을 해주는게 어떨까요?

Reply:
타입을 다음과 같이 사용하고 있습니다. `type ImagePathType = string;`를 추가 했습니다.
좀 더 직관적으로 타입을 선언 했어야 했는데...아쉽게 됐네요

```javascript
// types/index.ts
type ImagePathType = string;

export type UserImageType = {
  '160w': ImagePathType;
  '240w': ImagePathType;
  '320w': ImagePathType;
  '480w': ImagePathType;
  fallback: ImagePathType;
};

// UserViewSection3/index.tsx
import mobileView3_160w from '@/assets/mobileView3-160w.webp';
import mobileView3_240w from '@/assets/mobileView3-240w.webp';
import mobileView3_320w from '@/assets/mobileView3-320w.webp';
import mobileView3_480w from '@/assets/mobileView3-480w.webp';
import mobileView3_fallback from '@/assets/mobileView3-fallback.png';

const mobileView3: UserImageType = {
  '160w': mobileView3_160w,
  '240w': mobileView3_240w,
  '320w': mobileView3_320w,
  '480w': mobileView3_480w,
  fallback: mobileView3_fallback,
};

```

Reply:
mapped type을 이용하면 조금 더 제한적인 key 값을 부여할 수 있을 것 같아요.

```js
type ImagePathType = '160w' | '240w' | '320w' | '480w' | 'fallback';

type UserImageType = {[K in ImagePathType]: string};

const mobileView1: UserImageType = {
  '160w': mobileView1_160w,
  '240w': mobileView1_240w,
  '320w': mobileView1_320w,
  '480w': mobileView1_480w,
  fallback: mobileView1_fallback,
};
```